plasmapy/physics/tests/test_distribution.py

```diff
@@ -90,10 +93,16 @@ def velDist3D(vx, vy, vz):
                           lambda z: infApprox,
                           lambda z, y: -infApprox,
                           lambda z, y: infApprox,
-                          epsabs=1.49e-08*u.m/u.m, 
-                          epsrel=1.49e-08*u.m/u.m)
-    exceptStr = "Integral of distribution function should be 1."
-    assert np.isclose(integ.value, 1), exceptStr
+                          epsabs=0.0, 
```

Comment:
probably should change this from 0.0 to some larger value as I'm not sure how QUADPACK handles 0.0 absolute error (whether infinitely small or just ignored).